symbulate/distributions.py

```diff
@@ -618,16 +666,33 @@ def __init__(self,
                  sd1=1.0, sd2=1.0, corr=0.0,
                  var1=None, var2=None, cov=None):
 
-        if corr is not None and not (-1 <= corr < 1):
+        if corr is not None and not (-1 <= corr <= 1):
```

Comment:
corr cannot be None because its default value is 0.0. (Well, it can be if the user sets it to None, but why would any user do that?)

The only args that can be None are the ones whose default value is None.